recipes/calcam/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "calcam" %}
+{% set version = "2.13.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/euratom-software/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 6aa05c843253f0c2c2648354dd19158d556c472ef09edbbbd10ddf19b79525cd
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  entry_points:
+    - calcam = calcam:start_gui
+  number: 0
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - matplotlib-base
+    - opencv
+    - pyqt
+    - scipy
+    - vtk >=7,!=9.1.*
+    - h5py
+    - triangle
+  run_constrained:
+  # Note that libstdcxx-ng >=13 is required for VTK to function correctly.
+  # In my testing on Ubuntu + Anaconda , the default installe version is 11 and 
+  # this causes a crash at runtime (), whereas >=13 works properly.
+    - libstdcxx-ng >=13
```

Comment:
Then the `vtk` feedstock should be rebuilt. This is a `noarch` package and thus should not have any compiler specific entries.